try {
	hello();
} catch (error) {
	console.log("Failed");
}

try {
	anotherFunction();
} catch {
	console.log("I'm so empty");
}
